ksql-engine/src/test/java/io/confluent/ksql/structured/SqlPredicateTest.java

```diff
@@ -69,12 +72,12 @@ private PlanNode buildLogicalPlan(String queryStr) {
     analyzer.process(statements.get(0), new AnalysisContext(null));
     AggregateAnalysis aggregateAnalysis = new AggregateAnalysis();
     AggregateAnalyzer aggregateAnalyzer = new AggregateAnalyzer(aggregateAnalysis,
-        analysis);
+        analysis, ksqlFunctionRegistry);
```

Comment:
would make me happy to rename this to functionRegistry

Reply:
Definitely :) It's done!